internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -854,6 +860,28 @@ func (c *ClusterManager) buildObservabilityCloudwatchLoggingCredentialsSecret()
 	}
 }
 
+func (c *ClusterManager) buildObservabilityEnterpriseCloudwatchLoggingCredentialsSecret(orgId string, clusterId string) *k8sCoreV1.Secret {
```

Comment:
acronyms should be uppercased in Go unless they affect package visibility.
In this case it should be `orgID string, clusterID string`

Reply:
:+1: